src/navigation/deep-linker.ts

```diff
@@ -45,35 +28,84 @@ import { ViewController } from './view-controller';
  * Ionic developers to think of URLs as a breadcrumb rather than as the source of
  * truth in navigation. This encourages flexible navigation design and happy apps all
  * over the world.
- */
-
-/**
- * @private
  *
+ * Deep Linking Scenarios:
+ * 1. Initialize all NavControllers using the initial browser URL
```

Comment:
This was in there originally, didn't know if it should be removed. Will remove though